frontend/src/specific_components/file_data.rs

```diff
@@ -90,6 +90,9 @@ fn onchange_element_tree(element_tree: Rc<RefCell<ElementTree>>) -> Callback<Edi
 
 #[function_component]
 pub fn FileData(props: &Props) -> HtmlResult {
+    // TODO use backend::get_file here
```

Comment:
After that we should've factor file_data.rs and clean it up so we should not call yewdux from here but we should call it from backend::get_file because in the back and folder is the place where we handle the F a statement for the different environment web version, desktop and yewdux ..